section/storage.html

```diff
@@ -1,23 +1,167 @@
+            <h4>Authentication Keys</h4>
+            
+            <p>The {{AuthKeys}} dictionary contains the cryptographic key pair used for HTTP message signatures in Open Payments requests.</p>
+            
+            <pre class="idl">
+            dictionary AuthKeys {
+                required DOMString keyId;
+                required DOMString privateKey;
+                required DOMString publicKey;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>keyId</dfn></dt>
+                <dd>The key identifier used in HTTP signature headers to reference this specific key pair.</dd>
+                
+                <dt><dfn>privateKey</dfn></dt>
+                <dd>The private key for signing HTTP messages. Must be stored securely and encrypted.</dd>
+                
+                <dt><dfn>publicKey</dfn></dt>
+                <dd>The corresponding public key that can be shared with servers for signature verification.</dd>
+            </dl>
+        </section>
+        
+        <section data-dfn-for="PaymentRateInfo">
+            <h4>Payment Rate Information</h4>
+            
+            <p>The {{PaymentRateInfo}} dictionary contains payment rate configuration that controls the frequency and amount of Web Monetization payments.</p>
+            
+            <pre class="idl">
+            dictionary PaymentRateInfo {
+                DOMString rateOfPay;
+                DOMString maxRateOfPay;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>rateOfPay</dfn></dt>
+                <dd>The current rate of payment in the wallet's asset units per time interval (e.g., "1000" for 1000 units per second).</dd>
+                
+                <dt><dfn>maxRateOfPay</dfn></dt>
+                <dd>The maximum allowed rate of payment, acting as a safety limit to prevent excessive spending.</dd>
+            </dl>
+        </section>
+    </section>
+    
+    <section>
+        <h3>Storage Operations</h3>
+        
+        <div class="algorithm">
+            <p>To <dfn>get profile</dfn>:</p>
+            <ol>
+                <li>Let |profile| be the {{Profile}} instance from secure browser storage.</li>
+                <li>If no stored profile exists, return a new {{Profile}} with all properties set to null.</li>
+                <li>Return |profile|.</li>
+            </ol>
+        </div>
+        
+        <div class="algorithm">
+            <p>To <dfn>store wallet credentials</dfn>, given |walletAddressDetails:WalletAddressDetails|:</p>
+            <ol>
+                <li>Let |profile| be the result of [=get profile=].</li>
+                <li>Set |profile|'s {{Profile/wallet}} to |walletAddressDetails|.</li>
+                <li>[=Persist profile=] with |profile|.</li>
+            </ol>
+        </div>
+        
+        <div class="algorithm">
+            <p>To <dfn>store grant credentials</dfn>, given |grant:Grant|:</p>
```

Comment:
Let's reference this algorithm in "Connecting a wallet" section. We've an open `.issue` corresponding to this.